atlas-engine/AtlasEngine.FullstackServer
# Fix the Info Route when Running the AtlasEngine Inside a Docker Container

## Changes

1. Fix an issue where the `package.json` file was missing inside the directory, where the `InformationService` looks for it, when running the AtlasEngine inside a Docker container.

PR: #15 

## How to test the changes

* Rebuild the AtlasEngine Docker image
* Navigate to the `/atlas_engine/api/user_domain/v1/info` route
* See, that it returns the informations about the current atlas-engine version.
